python/semantic_kernel/connectors/ai/anthropic/services/utils.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import logging
+from enum import Enum
+
+from semantic_kernel.contents.utils.finish_reason import FinishReason as SemanticKernelFinishReason
+
+logger: logging.Logger = logging.getLogger(__name__)
+
+
+class AnthropicFinishReason(str, Enum):
```

Comment:
why do we need this, can't you just use the enum from the anthropic package? or just the string that is sent as the key for the dict below?

Reply:
Thanks, updated!